flw/src/main/resources/flw.properties

```diff
@@ -0,0 +1,7 @@
+#what time of day should the process of generating the targetFile start
+flw.purge_invalid_flw_start_time=06:00
+
+#one day = 1000ms x 60s x 60m x 24h = 86400000
+flw.purge_invalid_flw_ms_interval=86400000
```

Comment:
I was thinking of using the batch module for this, but ended up thinking it was overkill.  It might make sense to look into it again.

I do think having sub-day granularity is useful (at least for the data purge).  If there is a lot of data to purge running it twice a day could be useful.